awx-operator.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: awx-operator
+  version: 2.19.1
+  epoch: 0
+  description: An Ansible AWX operator for Kubernetes built with Operator SDK and Ansible.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ansible-operator
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - py3-ansible-core
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/ansible/awx-operator
+      tag: ${{package.version}}
+      expected-commit: dd37ebd440edf953d822f2c134833a44a8e77532
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}/opt/ansible
+      cp requirements.yml ${{targets.contextdir}}/opt/ansible/requirements.yml
+      HOME=${{targets.contextdir}}/opt/ansible ansible-galaxy collection install -r ${{targets.contextdir}}/opt/ansible/requirements.yml
+      cp watches.yaml ${{targets.contextdir}}/opt/ansible/watches.yaml
+      cp -r roles ${{targets.contextdir}}/opt/ansible/roles
+      cp -r playbooks ${{targets.contextdir}}/opt/ansible/playbooks
+
+update:
+  enabled: true
+  github:
+    identifier: ansible/awx-operator
+
+test:
+  pipeline:
+    - name: Check Ansible files
+      runs: |
+        stat /opt/ansible/requirements.yml
+        stat /opt/ansible/watches.yaml
+        stat /opt/ansible/roles
+        stat /opt/ansible/playbooks/awx.yml
+    - name: Smoke test run
+      runs: |
+        mkdir -p /var/run/secrets/kubernetes.io/serviceaccount
+        echo dummy >/var/run/secrets/kubernetes.io/serviceaccount/token
+        export KUBERNETES_SERVICE_HOST=localhost KUBERNETES_SERVICE_PORT=6443 KUBERNETES_MASTER=https://localhost:6443
```

Comment:
nb but are we simply invoking k8s env variable and not using our kwok test pipeline, is it important for this test to run?

Reply:
It's not vital because the controller will start anyway retrying to connect to the API server (we don't need to test the reconciliation), but that's a good point. I'd try to leverage it

Reply:
Addressed @mritunjaysharma394. Thank you! TIL